pages/common/postalias.md

```diff
@@ -0,0 +1,28 @@
+# postalias
+
+> Creates or updates postfix alias databses.
+> More information: <https://manned.org/postalias>.
+
+- Config dir read the main configuration file in the named directory:
+
+`postalias -c `
+
+- Do not fold the lookup key to lower case while creating a query table:
+
+`postalias -f`
+
+- Search the specified maps for key and remove one entry per map:
+
+`postalias -d {{key}}`
```

Comment:
```suggestion
- Search specific maps for key and [d]elete one entry per map:

`postalias -d "{{key}}"`
```

Optional fix: `specific` may sound better.
Note: we are discarding anything expect `specific` slowly.
Optional fix: mnemonic can be used.
Optional fix: placeholder can be quoted if it can contain special characters to POSIX-compatible shell.